### ISSUE TITLE
Focus is not visible for many UI controls

### ISSUE THREAD

**Issue by adampage** [lukevella/rallly#2420]
**Describe the bug**
Many UI controls do not show a visible focus indicator when they receive keyboard focus.

**To Reproduce**

1. Visit the [New Poll](https://web.local.rallly.co/new) page, as one example.
2. Use your keyboard to tab through the page’s focusable elements.
3. Observe that many do not show a visual change upon receiving focus (e.g., user menu, time of day dropdowns, etc.)

**Expected behavior**
All focused controls should show [visible focus](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html).

**Video capture**

[Video capture of tabbing through the page](https://drive.google.com/file/d/1Edr9y_dxCvlvg0cHkybcvLgmvmkFkY_W/view?usp=sharing)

**Desktop:**

- OS: macOS Tahoe 26.5
- Browser: Chrome
- Version: 48.0.7778.216
